### DISCUSSION TITLE
Does this repo subsume the docker handling in the core LFE?

### DISCUSSION THREAD

**Discussion by rvirding** [lfex/dockerfiles#28] [Q&A] [answered]
I am exposing my ignorance about docker here. Does this repo subsume the docker handling in the core LFE repo? Or do they complement each other so we need both?

**Comment by oubiwann** [comment]
An open source project that benefits from a docker image (to help bootstrap users) will often have one or two `Dockerfile`s in the main project. It's nice for users to see it associated with the project itself. It's typically the project owner's favorite/preferred Linux distro :-)

That's the benefit of having a `Dockerfile` in the project itself.

The benefit of having a repo like this one is that lots of different `Dockerfile`s can be provided without polluting the main project. Furthermore, these `Dockerfile`s provide additional features or cutting-edge features that haven't been merged to LFE's `develop` branch yet, or provide additional libraries and tools pre-installed that the LFE project may not want to do for its own `Dockerfile`.

**Comment by oubiwann** [comment]
This repo is now 100% based on the official Erlang docker images (which didn't exist at the time of this repo's creation). As such, there are only three images generated: 
* Debian-based full
* Debian-based slim
* Alpine-based

The default image is Alpine-based, since that's the smallest one. The Dockerhub tag for it is `lfex/lfe:latest` or just `lfex/lfe` and can be run with `docker run -it lfex/lfe`.

A new PR has just been submitted against the `Dockerfile` in the LFE repo which replaces its outdated contents simply with `FROM lfex/lfe:latest`, so it will never be out of sync with this repo.

**Comment by oubiwann** [comment, marked as answer]
I guess I never really explicitly answered you, @rvirding!

They compliment each other and community members will use both 😄
